### PR TITLE
DBZ-1639 apicurio demo with both converters

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-1639

### CODE REVIEW

File: tutorial/README.md [hunk 1]
```diff
@@ -80,6 +87,100 @@ docker-compose -f docker-compose-mysql-avro.yaml exec schema-registry /usr/bin/k
     --topic dbserver1.inventory.customers
 ```
 
+### Using MySQL and Apicurio Registry
+[Apicurio Registry](https://github.com/Apicurio/apicurio-registry) is an API/schema registry that could be used to store schemas of Kafka messages.
+The registry itself is an open source project that brings further additional features
+- The registry provides its own native API and an API compatible with Confluent's Schema registry. This allows using of already exisitng tools like Confluent's Avro converter for easier migration.
```
Comment (gunnarmorling) [inline tutorial/README.md]:
```suggestion
- The registry provides its own native API and an API compatible with Confluent's Schema Registry. This allows using of already existing tools like Confluent's Avro converter for easier migration.
```

File: tutorial/README.md [hunk 2]
```diff
@@ -80,6 +87,100 @@ docker-compose -f docker-compose-mysql-avro.yaml exec schema-registry /usr/bin/k
...
+- The registry provides its own native Avro converter and Protobuf serializer.
+- The registry provides a JSON converter that exports its schema into the registry.
+
+
+#### JSON format
+Configuring JSON converter with extrenalized schema at the Debezium Connector involves specifying the converter and schema registry as a part of the connectors configuration.
```
Comment (gunnarmorling) [inline tutorial/README.md]:
```suggestion
Configuring JSON converter with externalized schema at the Debezium Connector involves specifying the converter and schema registry as a part of the connectors configuration.
```

File: tutorial/README.md [hunk 3]
```diff
@@ -80,6 +87,100 @@ docker-compose -f docker-compose-mysql-avro.yaml exec schema-registry /usr/bin/k
...
+If you alter the structure of the `customers` table in the database and trigger another change event,
+a new version of that schema will be available in the registry.
+
+To consume the Avro messages It is possible to use `kafkacat` tool:
+
+```shell
+docker run --rm --tty \
+  --network tutorial_default \
+  debezium/tooling \
+  kafkacat -b kafka:9092 -C -o beginning -q -s value=avro -r http://apicurio:8080/confluent \
```
Comment (gunnarmorling) [inline tutorial/README.md]: Why is there "confluent" in the URL?

File: tutorial/debezium-with-apicurio/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+ARG DEBEZIUM_VERSION
+FROM debezium/connect:$DEBEZIUM_VERSION
+ENV KAFKA_CONNECT_DEBEZIUM_DIR=$KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-mysql
+
+COPY *.tar.gz $KAFKA_CONNECT_DEBEZIUM_DIR/apicurio.tar.gz
+
+RUN  cd $KAFKA_CONNECT_DEBEZIUM_DIR &&\
+    tar xvf apicurio.tar.gz &&\
+    rm apicurio.tar.gz
```
Comment (gunnarmorling) [inline tutorial/debezium-with-apicurio/Dockerfile]: Can you add a note to the README on this binary, its purpose and future plan to remove to upstream artifact?

File: tutorial/README.md (hunk 3 above)
Comment (jpechane) [reply tutorial/README.md]: @gunnarmorling That's path on which Confluent-compatible API is exposed

File: tutorial/debezium-with-apicurio/Dockerfile (hunk 4 above)
Comment (jpechane) [reply tutorial/debezium-with-apicurio/Dockerfile]: @gunnarmorling Would not it be better to comment directly in Dockerfile?

Comment (jpechane) [reply tutorial/debezium-with-apicurio/Dockerfile]: @gunnarmorling Commetn added to the Dockerfile

**Comment by jpechane** [comment]
@gunnarmorling Upgradet to apicurio 1.1.0.Final. The converter client still embedded as binary due to unknow location of Maven artifacts.

**Comment by jpechane** [comment]
@gunnarmorling I believe we are good to go, no binary artifacts any longer, both server and client are standard Apicurio

**Pull request merged**